src/eex_match/matchers/exact_matcher.py

```
@@ -25,94 +25,100 @@ class ExactMatcher(BaseMatcher):
         # Create signature lookup for exchange trades
         exchange_lookup = self._build_exchange_lookup(exchange_trades)
-        
+
         # Find matches
         for trader_trade in trader_trades:
             # Create matching signature for this trader trade
             signature = self._create_matching_signature(trader_trade)
-            
+
             # Look for matching exchange trades
             if signature in exchange_lookup and exchange_lookup[signature]:
```

Ruff (0.12.2)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)